internal/security/syscall_windows.go

```diff
@@ -2,6 +2,6 @@ package security
 
 //go:generate go run mksyscall_windows.go -output zsyscall_windows.go syscall_windows.go
```

Comment:
You can do `//go:generate go run golang.org/x/sys/windows/mkwinsyscall -output <> ...` do you dont have to include the `mksyscall_windows.go` file`